README.md

```diff
@@ -37,56 +37,41 @@ Quick Start
 #### Installation
 
 1. Install yaksh
-    - For latest stable release
+    - Clone the repository
 
-            $ pip install yaksh
+            $ git clone https://github.com/FOSSEE/online_test.git
 
-    - For the development version
+    - Go to the online_test directory
 
-            $ pip install git+https://github.com/FOSSEE/online_test.git
+            $ cd ./online_test
 
-#### Short instructions
-
-To see a quick demo after installing yaksh do the following:
+    - Install the dependencies
 
-    $ yaksh create_demo yaksh_demo
-    $ yaksh run yaksh_demo
+            $ pip install -r ./requirements/requirements-py3.txt
```

Comment:
Can we give an option to the user to install py3 or py2 packages and also mention that it is recommended to install requirements-py3.txt instead of blindly forcing the user  to go for py3?

Reply:
Also, I realized that the package **ruamel.yaml** requires the user to separately install setuptools. Could you add this to the requirements file. This was an error on my part.

Reply:
The requirements-py3.txt should either contain requirements-codeserver.txt or the command to install that should be in readme.

Reply:
Fixed